frontend_v2/src/styles/base.scss

```diff
@@ -1,14 +1,14 @@
 /*Add global styles here*/
 
-@import "./variables.scss";
-@import "./mixins.scss";
-@import "./grid.scss";
-@import "./font.scss";
-@import url("https://fonts.googleapis.com/css?family=Roboto:300,400,500,700");
-@import "./theming.scss";
+@import './variables.scss';
```

Comment:
By mistake changed the double quotes to single quotes in this PR. Will fix this in a follow up PR.